tests/ansible_collections/roles/update-system/tasks/update-system.yml

```
@@ -3,6 +3,10 @@
   yum:
     name: "*"
     state: latest
+    # In case the package is installed in the "artifact installation" phase during artemis
+    # provisioning method and the version of the package is not the latest in copr repo
```

```suggestion
    # In case the package is already installed on the system (by user or CI system) and the version of the package is not the latest in copr repo
```
I would be more generic, the case you described is not only one (and it is not completely correct)

---

Ok, thanks. I'm gonna change it/